include/afl-framework.h

```diff
@@ -0,0 +1,19 @@
+#include "afl-fuzz.h"
+#include "list.h"
+
+typedef struct afl_executor
+{
+    bool has_forkserver;
+    bool has_fauxserver;
+    afl_state_t * afl_state;
+    u32 map_size;
+} afl_executor_t;
+
+typedef struct executors
+{
+    list_t executors_list;
+    u8 number_of_executors;
```

Comment:
u8? I would propose u32. 256+ cores is a thing :)
plus maybe we will have remote executors

Reply:
Ohh. Ok. I'll change this. :slightly_smiling_face: